common/src/test/java/tachyon/util/io/FileUtilsTest.java

```diff
@@ -82,6 +88,16 @@ public void moveFileTest() throws IOException {
   }
 
   @Test
+  public void moveNonExistentFileTest() throws IOException {
+    // ghostFile is never created, so deleting should fail
+    File ghostFile = new File(mTestFolder.getRoot(), "ghost.txt");
+    File toFile = mTestFolder.newFile("to.txt");
+    mException.expect(IOException.class);
+    FileUtils.move(ghostFile.getAbsolutePath(), toFile.getAbsolutePath());
+    Assert.assertTrue(ghostFile.delete());
```

Comment:
I am going to replace the Assert.assertTrue assertions with Assert.fail(<informative message>) assertion to make it more obvious what the assertion is used for.